data/opcode_info.json

```diff
@@ -0,0 +1,1506 @@
+[
```

Comment:
It would be nice to have a comment at the top of this file like:

```
/** DO NOT EDIT
**  This file was generated by script/scrape_opcode_info.js
**/
```

Reply:
It would be, but alas, it is JSON and it does not allow comments.

Reply:
oh, duh.